DocType/AbstractDocType.py

```diff
@@ -16,11 +16,15 @@ def __init__(self, fileName):
 		Creates the structure for the file
 		"""
 		# Name of the original file
-		self.fileName = fileName
+		slash = fileName.rfind('/')
+		if slash != -1 :
```

Comment:
Why don't you use os.path methods dirname and basename ?
I just ran some tests and they return the following : 
````python
filePath = "./Texts/Dir2/myFile.txt"
os.path.basename(filePath)
#=> 'myFile.txt' 
os.path.dirname(filePath)
#=> './Texts/Dir2'
os.path.dirname("myFile.Txt")
#=> ''
````